service/src/main/kotlin/io/provenance/api/frameworks/web/external/provenance/ProvenanceApi.kt

```diff
@@ -380,7 +410,10 @@ class ProvenanceApi {
                 GET("/status", handler::getClassificationStatus)
                 GET("/fees", handler::getFees)
             }
-            GET("/scope/query", handler::queryScope)
+            "/scope".nest {
```

Comment:
Just a thought: should we re-organize `/permissions` and `/scope`? Either `/data` should move into the scope nesting, or `/scope` should move down to the permissions nesting.

Reply:
I think the former would be better

Reply:
Addressed in #61